src/gui/qopenglplotter.h

```diff
@@ -181,6 +181,7 @@ class QOpenGLPlotter : public QGLWidget
     void _deselect_all_sources();
 
   private:
+    qreal m_scale;
```

Comment:
Just a little note on coding style: In the rest of the SSR code, non-public members start with an underscore and not with `m_`. It would be more consistent to call this `_scale`.

https://github.com/SoundScapeRenderer/ssr/blob/6cadda022e69162871f466d12dcaebd208aa9252/src/coding_style.txt#L81-L85